apps/inheritance_app/inheritance_decrypt_data.c

```diff
@@ -298,7 +298,7 @@ static bool decode_inheritance_encrypted_data(
     uint16_t data_size,
     inheritance_decrypt_data_with_pin_encrypted_data_structure_t
         *encrypted_data) {
-  if (NULL == data || NULL == encrypted_data || 0 == data_size) {
```

Comment:
same here as described in `ecryption file`, but also in case of silver plan do we want to encrypt empty struct? i.e i assumed the value of plain_data_count from cysync would be zero so only the PIN would be encrypted, therefore removing the need of decrypting empty plain data at all.